client/src/components/Toasts/Toasts.scss

```diff
@@ -0,0 +1,160 @@
+.toasts {
+  position: fixed;
+  top: 60px;
+  right: 20px;
+  width: 250px;
+  z-index: 9999;
+
+  &__notice-wrap {
+    position: absolute;
+  }
+}
+
+
+@keyframes toastFadeIn {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+@keyframes toastFadeOut {
+  0% {
+    opacity: 1;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  50% {
+    opacity: 0;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  100% {
+    opacity: 0;
+    max-height: 0;
+    margin-bottom: 0;
+  }
+}
+
+
+.toast {
+  //height: 60px;
+  display: block;
+  position: relative;
+  margin: 0 0 map_get($spacers, sm) 0;
+  border-radius: 3px;
+  font-family: inherit;
+  font-size: inherit;
+  padding: 0;
+  word-wrap: break-word;
+  max-width: 250px;
+  max-height: 250px;
+  height: auto;
+  border: 0;
+  border-left: 3px solid;
+  color: #666;
+  left: 0;
+  background: #fff;
+  opacity: 1;
+  box-shadow: $z-depth-1;
+  transition: box-shadow $transition-speed-fast ease;
+  width: 250px;
+
+  &:not(:last-child) {
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  &:hover {
+    box-shadow: $z-depth-2;
+  }
+  animation: $transition-speed-mid ease-out 0s 1 toastFadeIn;
+
+  &--success,
+  &--good,
+  &--green {
+    border-color: $color-good;
+  }
+
+  &--notice,
+  &--info,
+  &--blue {
+    border-color: $color-notice;
+  }
+
+  &--warning,
+  &--caution,
+  &--yellow {
+    border-color: $color-warning;
+  }
+
+  &--bad,
+  &--error,
+  &--red {
+    border-color: $color-error;
+  }
+
+  &--dismissing {
+    animation: $transition-speed-mid*2 ease-out 0s 1 toastFadeOut;
```

Comment:
Is `$transition-speed-mid*2` an arbitrary value? Maybe you could introduce `$transition-speed-extra-long`? 😅

Reply:
I multiplied the value by a factor of 2 because there's two steps to the transition:
- fading out by transitioning opacity from `1` to `0`
- sliding up the toast notification below by reducing the max-height of the toast from `250px` to `0`.

I want both of those animation to take `$transition-speed-mid` seconds to complete. There is at least a little bit of logic to my madness.

Reply:
Could there be a comment added to describe why you've done this for future reference?

Reply:
Makes sense @maxime-rainville, I agree with Sacha on adding a note about it, does look quite arbitrary at a glance.

Reply:
I added a blurb.